src/mixins/dropdown.js

```diff
@@ -84,7 +86,9 @@ export default {
     return {
       visible: false,
       inNavbar: null,
-      visibleChangePrevented: false
+      visibleChangePrevented: false,
+      listenForClickOut: false,
+      listenForFocusIn: false
```

Comment:
Since these last two data items are defined in the mixins, you could leave them out (or comment them out and leave in noting they are in the mixins). When the mixins are imported and added, the data will be merged with the component data.

Reply:
They are set to true in the mixin by default and I deactivate them here because we want them only to be active when the dropdown is open.

Reply:
Ah, true...

Reply:
Him I wonder if the mixing should default them to false, and then any component that uses the mixins could set the values on created or mounted hook?